File: backend/dotnet/ticketflow/TicketFlow.MovieService/Domain/Entities/Film.cs
using System;

namespace TicketFlow.MovieService.Domain.Entities
{
    internal class Film : IFilm
    {
        public Film(int id, string title, string description, DateTime premiereDate, string creator, int duration, int ageLimit)
        {
            Id = id;
            Title = title;
            Description = description;
            PremiereDate = premiereDate;
            Creator = creator;
            Duration = duration;
            AgeLimit = ageLimit;
        }

        public int Id { get; }

        public string Title { get; }

        public string Description { get; }

        public DateTime PremiereDate { get; }

        public string Creator { get; }

        public int Duration { get; }

        public int AgeLimit { get; }
    }
}

File: backend/java/movie_service/src/main/java/com/ticketflow/movie_service/models/Film.java
package com.ticketflow.movie_service.models;

import java.time.LocalDate;

public class Film {
    private Integer id;
    private String title;
    private String description;
    private LocalDate premiereDate;
    private String creator;
    private Integer duration;
    private Integer ageLimit;

    public Film(
            Integer id,
            String title,
            String description,
            LocalDate premiereDate,
            String creator,
            Integer duration,
            Integer ageLimit) {
        this.id = id;
        this.title = title;
        this.description = description;
        this.premiereDate = premiereDate;
        this.creator = creator;
        this.duration = duration;
        this.ageLimit = ageLimit;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public LocalDate getPremiereDate() {
        return premiereDate;
    }

    public void setPremiereDate(LocalDate premiereDate) {
        this.premiereDate = premiereDate;
    }

    public String getCreator() {
        return creator;
    }

    public void setCreator(String creator) {
        this.creator = creator;
    }

    public Integer getDuration() {
        return duration;
    }

    public void setDuration(Integer duration) {
        this.duration = duration;
    }

    public Integer getAgeLimit() {
        return ageLimit;
    }

    public void setAgeLimit(Integer ageLimit) {
        this.ageLimit = ageLimit;
    }
}